software/conduwuit.yml

```
@@ -0,0 +1,10 @@
+name: conduwuit
+website_url: https://conduwuit.puppyirl.gay
+description: conduwuit is a well-maintained, featureful, hard-fork of Conduit, which is a Matrix chat homeserver.
```

```suggestion
description: Well-maintained, featureful Matrix chat homeserver (fork of Conduit).
```

https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/CONTRIBUTING.md#other-guidelines

---

https://conduwuit.puppyirl.gay/differences.html @nodiscc

---

Also applied the suggested description.

---

https://conduwuit.puppyirl.gay/differences.html is more than fine, thank you, I did not see it at first.